Support for alternative container runtimes - Podman (#2745)
dapr/docs: daprdocs/content/en/reference/cli/dapr-init.md

|        N/A            |DAPR_HELM_REPO_URL|          | Specifies a private Dapr Helm chart url|
|        N/A            | DAPR_HELM_REPO_USERNAME | A username for a private Helm chart | The username required to access the private Dapr Helm chart. If it can be accessed publicly, this env variable does not need to be set|
|        N/A            | DAPR_HELM_REPO_PASSWORD | A password for a private Helm chart  |The password required to access the private Dapr Helm chart. If it can be accessed publicly, this env variable does not need to be set| |

> |  `--container-runtime`  |              |    docker      | It is used to pass in a different container runtime other than docker. Supported container runtime are docker/podman|

Reviewer: ```suggestion
|  `--container-runtime`  |              |    `docker`      | Used to pass in a different container runtime other than Docker. Supported container runtimes are: `docker`, `podman` |
```

2. Redis container image(rejson)   - docker.io/username/dapr/3rdparty/rejson
3. zipkin container image(zipkin)  - docker.io/username/dapr/3rdparty/zipkin

> You can specify a different container runtime while setting up dapr. If you omit this flag then default container runtime is docker.

Reviewer: ```suggestion
You can specify a different container runtime while setting up Dapr. If you omit the `--container-runtime` flag, the default container runtime is Docker.
```
